scriptmodules/emulators/jzintv/01_rpi_pillar_boxing_black_background_sdl2.patch

```diff
@@ -0,0 +1,15 @@
+--- a/gfx/gfx_sdl2.c
```

Comment:
Is this a SDL bug ? Then perhaps should be fixed in SDL repo (retropie/sdl).

Reply:
No it is how the emulator calls/uses SDL. SDL works right. The emulator assumes two things, and if both are false the issue (colored pillars) do show up. One: Physical display is always 4:3, two: The ROM does use black as background color.